docs: fix linter errors for accessibility.md (#47725)
angular/angular: aio/content/guide/accessibility.md

### Active links identification

> CSS classes applied to active `RouterLink` elements (usually via `RouterLinkActive`) provide a visual guide to which link is currently active. This guide doesn't apply to blind or visually impaired users. To provide such information the `aria-current` attribute should be applied to the element as well (for more information see [Mozilla Developer Network (MDN) aria-current](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current)).

Reviewer: This is a pretty dense sentence. I tried to split the ideas out here. If this conveys the same information, you can use this change. If not, please revise as required.
```suggestion
CSS classes applied to active `RouterLink` elements, such as `RouterLinkActive`, provide a visual cue to identify the active link.
Unfortunately, a visual cue doesn't help blind or visually impaired users.
Applying the `aria-current` attribute to the element can help identify the active link.
For more information, see [Mozilla Developer Network (MDN) aria-current](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current)).
```

    </nav>
```

<!-- vale Angular.Angular_Spelling = NO -->

> ## Extra resources

Reviewer: ```suggestion
## More information
```
